app/src/main/java/com/android/voyageur/model/user/User.kt

```diff
@@ -0,0 +1,36 @@
+package com.android.voyageur.model.user
+
+import com.android.voyageur.model.trip.Trip
+import java.time.Instant // For timestamp handling
+
+data class User(
+    val id: String,
+    var name: String = "",
+    var email: String = "",
+    var profilePicture: String = "",
+    var bio: String = "",
+    private val _trips: MutableList<Trip> = mutableListOf(),
+    private val _friends: MutableList<User> = mutableListOf()
+) {
+    val trips: List<Trip>
```

Comment:
What is this used for?

 Also, a "friends" feature doesn't make sense in our case.

Reply:
I understand your point, but I think including a "friends" feature in our User model could be beneficial for future development. Even if we don't plan to implement it immediately, having the groundwork in place might make it easier to add social features down the line. This could enhance the user experience by fostering a sense of community and engagement within the app.